Assignment2/Sixtrax-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,55 @@
+{
+	"@context":{
+			"class":"http://www.example.org/Class#",
+      		"element":"http://www.example.org/Elements#",
+			"pr":"http://www.example.org/Properties#",
+      		"msr":"http://www.example.org/Measurements#",
+      		"user":"http://www.example.org/Users#",
+			"xsd":"http://www.w3.org/2001/XMLSchema#",
+			"Class01":"class:Class01",
+      		"includes":"pr:includes",
+			"hasMeasurement":"pr:hasMeasurement",
+			"hasOwner":"pr:hasOwner",
+			"hasTemperature":"pr:hasTemperature",
+			"atTime":"pr:atTime",
+			"hasName":"pr:hasName",
+			"Sensor029":"element:Sensor029",
+			"Computer101":"element:Computer101",
+			"Measurement8401":"msr:Measurement8401",
+			"User10A":"user:User10A"
+	},
+		"@id":"http://www.example.org/Class01",
+		"@type":"class:Class01",
+		"includes":[{
+			"@id":"element:Sensor029/",
+			"@type":"element",
+          	"hasMeasurement":{		
+				"Measurement8401":{

```

Comment:
You don't need "Measurement8401":  It is the same case as I mentioned to you before for the "Sensor029":